FoxHound.Client/src/AddBlog.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, ChangeEvent, FormEvent } from "react";
+import axios from "axios";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import Container from "@material-ui/core/Container";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+
+interface IProps {
+  refreshBlogList: () => void;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2, 3),
+  },
+}));
+
+const AddBlog: React.FC<IProps> = (props) => {
+  const [owner, setOwner] = useState<string>("");
+  const [title, setTitle] = useState<string>("");
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [error, setError] = useState<string>("");
+  const { refreshBlogList } = props;
+
+  useEffect(() => {}, []);
```

Comment:
Can be removed

Reply:
1. I am using setError within the try/catch for Adding the new blog.
2. 'refreshBlogList' constant could be removed and referenced below as props.refreshBlogList but I did it this way for consistency.
3. useEffect was not used, I will remove.

Reply:
Sorry, i just tried highlighting useEffect...still getting used to this PR experience.

Reply:
no problem, just wanted to make sure I didn't miss a change.